tools/tasks/build.bundles.app.ts

```diff
@@ -0,0 +1,21 @@
+import {join} from 'path';
+import * as Builder from 'systemjs-builder';
+import {
+  JS_PROD_APP_BUNDLE,
+  JS_DEST,
+  SYSTEM_BUILDER_CONFIG,
+} from '../config';
+
+const BUNDLER_OPTIONS = {
+  minify: true,
+  mangle: false
```

Comment:
Would you bundle to CommonJS (`format: "cjs")? I think it will reduce the size of the bundle and ease the configuration.

It will also require [some shims](https://github.com/systemjs/builder/issues/319).